packages/roosterjs-editor-dom/lib/utils/preprocessTable.ts

```diff
@@ -0,0 +1,47 @@
+import normalizeRect from './normalizeRect';
+
+/**
+ * Preprocess the HTML content of the table so that we can resize it properly:
+ *     1. Remove useless width/height HTML attributes (presentational attributes) as they will be overriden by css properties
+ *     2. Convert pt to px if there is any
+ *     3. If unit used is neither px nor pt, or, no unit specified, caculate it based on bounding rectangle
+ */
+
+const Pt2PxCoefficient = 1.333;
+
+export default function preProcessTable(table: HTMLTableElement) {
+    if (table.hasAttribute('width')) {
```

Comment:
what if the table only has "width" and no cells has "width"? remove this attribute will make the table looks smaller (or bigger)

Reply:
And when you want to remove an attribute, no need to check if "hasAttribute", just remove it directly.